eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -9,49 +9,49 @@ parameters:
   BuildDocs: true
   TestMatrix:
     Linux_Python27:
-      OSName: 'Linux'
+      OSVmName: 'Linux'
```

Comment:
Once we resolve https://github.com/Azure/azure-sdk-tools/pull/610#discussion_r424061807 we should be able to simply remove the OSName from all the matrix entries.